lib/dues_calculator.rb

```diff
@@ -1,15 +1,9 @@
 # frozen_string_literal: true
 
 module DuesCalculator
-  def self.dues_for_n_competitors(country_iso2, base_entry_fee_lowest_denomination, currency_code, n)
+  def self.dues_per_competitor(country_iso2, base_entry_fee_lowest_denomination, currency_code)
     dues_per_competitor_in_usd_money = dues_per_competitor_in_usd(country_iso2, base_entry_fee_lowest_denomination, currency_code)
-    if dues_per_competitor_in_usd_money.present?
-      (dues_per_competitor_in_usd_money * n).exchange_to(currency_code)
-    else
-      nil
-    end
-  rescue CurrencyUnavailable
-    nil
```

Comment:
I think this exception will still happen in local development when using the `EuCentralBank` exchange rates, so you should not delete the `rescue` part.

Reply:
Okay added back. This method is used only for the competition edit page, and I was thinking to show error there in case the currency is not available. Showing 0 dues is also fine, so I've added it back.